web_server/social_distribution/settings.py

```diff
@@ -43,15 +43,16 @@
     'posts.apps.PostsConfig',
     'comments.apps.CommentsConfig',
     'friendship.apps.FriendshipConfig',
-    'users.apps.UsersConfig'
+    'users.apps.UsersConfig',
+    'crispy_forms'
```

Comment:
Crispy forms seems to be library you are using? Why? Do we have a place to document this so if the front end people try using something else we dont conflict?